test/mock-server/jwt.ts

```diff
@@ -29,4 +29,12 @@ const header: Header = {
   alg: 'HS256',
 }
 
-export const generateToken = () => create(header, payload, '')
+export const generateToken = async () => {
+  const key = await crypto.subtle.generateKey(
+    { name: 'HMAC', hash: 'SHA-256' },
+    true,
+    ['sign', 'verify']
+  )
```

Comment:
Why is this needed?

Reply:
It's an update from `djwt` (https://github.com/timonson/djwt). Before this, the encryption function took a `string` key. But from the new versions, it uses `CryptoKey` which need to be initialised this way